internal/pkg/aws/ecr/ecr.go

```diff
@@ -167,9 +168,30 @@ func (s Service) DeleteImages(images []Image, repoName string) error {
 	return nil
 }
 
+func (s Service) repoExist(repoName string) (bool, error) {
+	_, err := s.ecr.DescribeRepositories(&ecr.DescribeRepositoriesInput{
+		RepositoryNames: aws.StringSlice([]string{repoName}),
+	})
+	if err != nil {
+		aerr := err.(awserr.Error)
+		if aerr.Code() != "RepositoryNotFoundException" {
```

Comment:
Verify the cast succeeded with:
```go
aerr, ok := err.(awserr.Error)
if !ok {
...
```

Otherwise `aerr.Code()` could panic.

Reply:
Thought about that question but I thought the error returns from calling `DescribeRepositories` which should be always expected to return an `awserr`? Anyway there's no downside for adding it so why not.

Reply:
Yeah that may be true today, but that could change. The cast check is a good safety measure for that.